x-pack/plugins/secops/public/components/page/hosts/uncommon_process_table/index.tsx

```diff
@@ -190,42 +190,45 @@ const getUncommonColumns = (startDate: number): Array<Columns<UncommonProcessesE
     hideForMobile: false,
     render: ({ node }) => {
       const hosts: HostEcsFields[] = node.host;
-      const draggables = hosts.map(({ id, name }) => {
-        if (name != null && id != null) {
+      const draggables = hosts
+        .filter(({ id, name }) => id != null && name != null)
```

Comment:
Just noting, if `name` is provided but `id` is not, no host will be listed. This is different behavior than in other areas of the codebase like [Authentications Table](https://github.com/elastic/kibana/blob/feature-secops/x-pack/plugins/secops/public/components/page/hosts/authentications_table/index.tsx#L247-L254) where it'll still display the host name, just not make it an EuiLink to host details (since it needs the id for that).

As you mentioned elsewhere:
> It gets tricky if we are making the schema looser in a sense where maybe we didn't pull back at least an `id/name` pair to the front end or one or the other happens to be missing.

Feel free to leave this as is since we still have ambiguity here, but I'll make a note in the tech debt issue for creating a HOC for `HostDetails` links that way we'll have consistent behavior across the app in these situations.